Dungeon-Escape-Survival-Alpha.py

```diff
@@ -1,7 +1,7 @@
 import time, random, webbrowser
 #----Above this line is our imports.----
 groupname = "\nEvil Wild Eye Games\n"
-gamebuild = "\nCurrent Build: A-0.0.107"
+gamebuild = "\nCurrent Build: A-0.0.106"

```

Comment:
This should be gamebuild = "\nCurrent Build: A-0.0.108" , not gamebuild = "\nCurrent Build: A-0.0.106"